CodingChallenges/CC_034_DLA/Processing/CC_034_DLA/CC_034_DLA.pde

```diff
@@ -0,0 +1,62 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
+// Code for this video: https://youtu.be/Cl_Gjj80gPE
+// Processing transcription: Chuck England
+
+import java.util.*;
+
+List<Walker> tree = new ArrayList<Walker>();
+List<Walker> walkers = new ArrayList<Walker>();
+//final float r = 4;
+final int maxWalkers = 50;
```

Comment:
`final` is not typically used in Processing examples. Though I'm ok with keeping it, it's a minor detail and perhaps nice for viewers to learn about!

Reply:
Well, it can be confusing to new folks. I'll remove it.